scala/lambdas/dynamo-formatters/src/main/scala/uk/gov/nationalarchives/dynamoformatters/DynamoReadUtils.scala

```diff
@@ -61,7 +67,7 @@ class DynamoReadUtils(folderRowAsMap: Map[String, AttributeValue]) {
     getPotentialListOfValues(originalMetadataFiles, convertListOfStringsToT(UUID.fromString)),
     getNumber(sortOrder, _.toInt),
     getNumber(fileSize, _.toLong),
-    getValidatedMandatoryFieldAsString(checksumSha256),
+    checksums,
```

Comment:
Shouldn't we validate this as it could be empty which would be bad. @MancunianSam, is the reason `identifiers` doesn't have validation because it's fine if there are 0 identifiers or because the identifiers can vary, making it hard to validate?

Reply:
hmmmm that is a good point, let me rethink this

Reply:
@techncl Yep, identifiers doesn't need validation because there can be zero identifiers.